projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -16,9 +16,10 @@
  */
 
 import { Component, OnInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
 import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
 import { Observable } from 'rxjs';
-import { TranslateService } from '@ngx-translate/core';
+import { RecordSearchResultComponent } from '@rero/ng-core/lib/record/search/result/record-search-result.component';
```

Comment:
This import is wrong. It must be imported from ng-core public-api: 

```
import { RecordSearchResultComponent } from '@rero/ng-core'
```

Reply:
Unused import, actually. Removed.